runtime/calamari/src/lib.rs

```diff
@@ -173,12 +173,20 @@ parameter_types! {
     pub const SS58Prefix: u8 = manta_primitives::constants::CALAMARI_SS58PREFIX;
 }
 
+parameter_types! {
+    pub UnpausablePallets: Vec<Vec<u8>> = vec![b"Democracy".to_vec(), b"Balances".to_vec(), b"Council".to_vec(), b"CouncilCollective".to_vec(), b"TechnicalCommittee".to_vec(), b"TechnicalCollective".to_vec()];
```

Comment:
`UnpausablePallets` should be `NonPausablePallets` otherwise it can be confused with `UnpauseOrigin`

Reply:
renamed.